script/bootstrap

```diff
@@ -7,4 +7,4 @@ set -e
 cd "$(dirname "$0")/.."
 
 echo "Installing test dependencies..."
-python3 -m pip install tox colorlog pre-commit
+python3 -m pip install tox colorlog pre-commit mypy==0.740
```

Comment:
How about this
```suggestion
python3 -m pip install tox colorlog pre-commit $(grep mypy requirements_test.txt)
```

Reply:
I think that this could work. 

@scop what do you think ?

Reply:
Yeah, can't think of a better way right now.

On the other hand, I was about to suggest that for installing test dependencies, we should... well install requirements_test.txt. But what script/bootstrap does is not install test dependencies like it says, more like development dependencies. Will submit a tiny PR to fix that.